schema/draft/schema.json

```
@@ -561,43 +615,14 @@
                     "type": "string"
                 },
                 "params": {
-                    "properties": {
-                        "message": {
-                            "description": "The message to present to the user.",
-                            "type": "string"
+                    "anyOf": [
```

```suggestion
                    "oneOf": [
```

Why `anyOf`? I think we want it to be strictly one or the other. On the other hand, I get confused with the nitty-gritty differences between these two combinators.

---

There are other `anyOf` vs `oneOf` in the tweaks to the json-schema where this same question is relevant.